document_page/models/document_page.py

```diff
@@ -33,20 +35,12 @@ class DocumentPage(models.Model):
         required=True,
     )
 
-    # no-op computed field
     draft_name = fields.Char(
-        string="Name",
-        help="Name for the changes made",
-        compute=lambda x: x,
-        inverse=lambda x: x,
+        string="Name", help="Name for the changes made", store=False
```

Comment:
Why not store it?

Reply:
If we don't want it to be a store=False field, it wouldn't be better if it is related="history_head.name" with readonly=False.

Reply:
Please tell me the goal of this field to decide.

Reply:
The goal of the draft_name field is to give a value to the name field of the document.page.history objet that is created when the content of a document.page is modified

Reply:
Then why it shouldn't be stored?